cmd/gokr-packer/packer.go

```diff
@@ -410,6 +426,32 @@ func logic() error {
 		filename: "ca-bundle.pem",
 		fromHost: cacerts,
 	})
+
+	deployCertFile, deployKeyFile, err := getCertificate()
+	if err != nil {
+		return err
+	}
+	schema := "http"
+	if deployCertFile != "" {
+		// User requested TLS
+		schema = "https"
+		if *tlsInsecure {
```

Comment:
Take a look at this modification of mine here: this makes upgrading a little smoother.

We can now use `gokr-packer -tls=self-signed -insecure` to upgrade from HTTP to HTTPS, and removing the `-tls` flag reverts back to HTTP.